probe-builder/build-probe-binaries

```diff
@@ -403,7 +399,34 @@ function build_probe {
 		-e HASH=$HASH \
 		-e HASH_ORIG=$HASH_ORIG \
 		--name $CONTAINER_NAME \
-		$IMAGE_NAME
+		$IMAGE_NAME || return 1
+}
+
+function build_probe {
+	PROBE_ID=$PROBE_NAME-$PROBE_VERSION-$ARCH-$KERNEL_RELEASE-$HASH.ko
+
+	# Skip Kernel 4.15.0-29 because probe does not build against it
+	if [ $KERNEL_RELEASE-$HASH = "4.15.0-29-generic-ea0aa038a6b9bdc4bb42152682bba6ce"  ]; then
+		echo "Temporarily skipping $PROBE_ID"
+		return
+	fi
+
+	if [ -f $BASEDIR/output/$PROBE_NAME-$PROBE_VERSION-$ARCH-$KERNEL_RELEASE-$HASH.ko ] &&
+	   [ -f $BASEDIR/output/$PROBE_NAME-$PROBE_VERSION-$ARCH-$KERNEL_RELEASE-$HASH_ORIG.ko ]; then
```

Comment:
I'm not familiar with this syntax. I assume it's valid. I thought you would do:
```
if [ -f FILE1 -a -f FILE2 ]; then
```

If these are equivalent then no worries

Reply:
basically same thing, `-a` is handled by `[`, `&&` is handled by the shell. In any case, it's existing code moved between functions, nothing new.